tarpc/src/server.rs

```diff
@@ -66,6 +68,27 @@ impl Config {
     }
 }
 
+/// A [`Stub`] implementation that simply warps a `Serve`.
+pub struct ServeStub<S> {
```

Comment:
Maybe keep this type private for now and return an imp Stub? Can always make it public later if needed.

Reply:
Good idea